client/jetpack-connect/plans.jsx

```diff
@@ -71,7 +71,7 @@ class Plans extends Component {
 		if ( this.props.selectedPlan ) {
 			this.selectPlan( this.props.selectedPlan );
 		}
-		if ( this.props.hasPlan || this.props.notJetpack ) {
+		if ( this.props.hasPlan === true || this.props.notJetpack === true ) {
```

Comment:
It makes sense to be strict with falsy ones because of differentiating between `null`, `undefined` and `false`, but what does the `=== true` addition change, and in which cases would these stricter truthfullness checks be useful?